src/libsystem/CMakeLists.txt

```diff
@@ -75,6 +75,7 @@ target_link_libraries(system PRIVATE
 	corecrypto
 	commonCrypto
 	system_dnssd
+	system_networkextension
```

Comment:
I don't understand why but for some reason, darling is not able to find `/usr/lib/system/libsystem_networkextension.dylib`.

Reply:
Look into cmake/use_ld64.cmake

Reply:
That was the issue, thanks!